src/xcode/ENA/ENA/Source/AppDelegate & Globals/Route.swift

```diff
@@ -15,7 +15,7 @@ enum Route {
 		}
 		self.init(url: url)
 	}
-	// swiftlint:disable:next cyclomatic_complexity
+	// swiftlint:disable:next cyclomatic_complexity superfluous_disable_command
```

Comment:
```suggestion
	// swiftlint:disable:next cyclomatic_complexity
```
Seems like its still needed, so no superfluous command there anymore